### ISSUE TITLE
any chances for a flatpak version

### ISSUE THREAD

**Issue by btreut** [tectonic-typesetting/tectonic#140]
is there any chance for a flatpak Installation package?

see: http://flatpak.org

**Comment by pkgw** [comment]
Yes! I am not sure when I will have the time to do this, but it making a flatpak package should be straightforward.

**Comment by btreut** [comment]
thanks in advance
